site/src/pages/playground.js

```diff
@@ -1,21 +1,15 @@
 import React, { useState } from 'react';
 import Layout from '@theme/Layout';
+import Editor from '@monaco-editor/react';
```

Comment:
Not really happy about this editor. It loads from a CDN, seems a bit hard to understand how to get it to run from our own instance